### PR TITLE
upload

### PR DESCRIPTION
**Pull request by iush12**
done
Slide 4_3 - 5
1ae2be8509f9274d15f1c331d1c6b1a6
87a15d971d2df29e84c2e1ad5af25ff8
319561001_5789165721129872_3552136692749425485_n
banner-lg
cd038e3e8e2e7971a4169e15040e4b11
e2bb88afa2fcf015aca8e97fdadc8d1a
e465cf53ab42c5644c2d352059c0f0e1
fe56ddb679eda9afd26ff9aca3c42dbe
Frame 5
Slide 4_3 - 3
Slide 4_3 - 3
Slide 4_3 - 4
Slide 4_3 - 4
Slide 4_3 - 5
Slide 4_3 - 5
Slide 4_3 - 7
Slide 4_3 - 7
Slide 16_9 - 1

### CODE REVIEW

**Review by pallavi000 — APPROVED** [review]

**Review by pallavi000 — APPROVED** [review]

File: input.css [hunk 1]
```diff
@@ -94,4 +94,4 @@ left: 399px;
 
 
 
-}
+}
```
Comment (iush12) [inline input.css:97]: done

File: index.html [hunk 2]
```diff
@@ -43,5 +43,5 @@
 <div class="cc"><img src="e465cf53ab42c5644c2d352059c0f0e1.jpg" alt=""></div>
 <img src="fe56ddb679eda9afd26ff9aca3c42dbe.jpg" alt=""></div>
 </div>
-
+<div class="contact">Contact me</div>
```
Comment (iush12) [inline index.html:46]: m

File: index.html [hunk 3]
```diff
@@ -26,7 +26,7 @@
 <div class="d "><img src="Slide 4_3 - 7.jpg"></div></div>
 
 
-</div></div></br></br>
+</div></div>
```
Comment (iush12) [inline index.html:29]: h

**Comment by iush12** [comment]
nn

**Review by iush12 — COMMENTED** [review]
done

**Pull request merged**